lib/tasks/webpacker/check_node.rake

```diff
@@ -0,0 +1,25 @@
+namespace :webpacker do
+  desc "Verifies if webpacker is installed"
+  task :check_node do
+    puts "Checking node version..."
+    node_version = `node -v`
+
+    unless $?.success?
+      puts "Node.js not installed"
```

Comment:
Thanks for this. Suggestion:

```
 puts "Node is not installed" #Use Node since all instructions mention node instead of nodejs
 puts "Please install node at https://nodejs.org/en/download/ and run bundle exec rails webpacker:install"
 exit!
```